_security-plugin/configuration/ldap.md

```diff
@@ -537,3 +537,106 @@ authz:
           - 'cn=Jane Doe,ou*people,o=TEST'
           - '/\S*/'
 ```
+
+### (Advanced) Configuring multiple user and role bases
+
+If you need to configure multiple user bases in authc and/or authz section, this can be done using below syntax:
```

Comment:
To configure multiple user bases in authc and/or authz section, use the following syntax:

Reply:
Done